cunumeric/sphinxext/_comparison_config.py

```diff
@@ -0,0 +1,479 @@
+    "sometrue",
+    "test",
+}
+
+# these do not have valid intersphinx references
+MISSING_NP_REFS = {
+    "numpy.loads",
+    "numpy.mafromtxt",
+    "numpy.ndarray.flat",
+    "numpy.ndarray.shape",
+    "numpy.ndarray.T",
+    "numpy.ndfromtxt",
+}
+
+
+@dataclass(frozen=True)
+class SectionConfig:
+    title: str
+    attr: str | None
+    types: tuple(type) | None = None
+    names: tuple(str) | None = None
+
+
+FUNCTIONS = (FunctionType, BuiltinFunctionType)
+METHODS = (MethodType, MethodDescriptorType)
+UFUNCS = (numpy.ufunc,)
+
+NUMPY_CONFIGS = [
+    SectionConfig("Module-Level", None, types=FUNCTIONS),
+    SectionConfig("Ufuncs", None, types=UFUNCS),
+    SectionConfig("Multi-Dimensional Array", "ndarray", types=METHODS),
+    SectionConfig("Linear Algebra", "linalg", types=FUNCTIONS),
+    SectionConfig("Discrete Fourier Transform", "fft", types=FUNCTIONS),
+    SectionConfig("Random Sampling", "random", types=FUNCTIONS),
+]
+
+CONVOLVE = ("convolve", "correlate")
+
+LOGICAL = (
+    "all",
+    "allclose",
+    "any",
+    "array_equal",
+    "array_equiv",
+    "equal",
+    "greater_equal",
+    "greater",
+    "isclose",
+    "iscomplex",
+    "iscomplexobj",
+    "isfinite",
+    "isfortran",
+    "isinf",
+    "isnan",
+    "isnat",
+    "isneginf",
+    "isposinf",
+    "isreal",
+    "isrealobj",
+    "isscalar",
+    "less_equal",
+    "less",
+    "logical_and",
+    "logical_not",
+    "logical_or",
+    "logical_xor",
+    "not_equal",
+)
+
+EINSUM = (
+    "dot",
+    "einsum",
+    "inner",
+    "matmul",
+    "outer",
+    "tensordot",
+    "trace",
+)
+
+SET = (
+    "in1d",
+    "intersect1d",
+    "isin",
+    "setdiff1d",
+    "setxor1d",
+    "union1d",
+    "unique",
+)
+
+MANIP = (
+    "flat",
+    "flatten",
+    "ravel",
+    "repeat",
+    "reshape",
+    "reshape",
+    "resize",
+    "shape",
+    "squeeze",
+    "swapaxes",
+    "T",
+    "transpose",
+)
+
+FACTOR = ("cholesky", "qr")
+
+SVD = ("lstsq", "matrix_rank", "pinv", "svd")
+
+EIGEN = ("eig", "eigh", "eigvals", "eigvalsh")
+
+LU = (
+    "det",
+    "inv",
+    "pinv",
+    "slogdet",
+    "solve",
+    "tensorinv",
+    "tensorsolve",
+)
+
+CREATION = (
+    "arange",
+    "array",
+    "asanyarray",
+    "asarray",
+    "ascontiguousarray",
+    "asmatrix",
+    "bmat",
+    "copy",
+    "diag",
+    "diagflat",
+    "empty_like",
+    "empty",
+    "eye",
+    "frombuffer",
+    "fromfile",
+    "fromfunction",
+    "fromiter",
+    "fromstring",
+    "full_like",
+    "full",
+    "geomspace",
+    "identity",
+    "linspace",
+    "loadtxt",
+    "logspace",
+    "mat",
+    "meshgrid",
+    "mgrid",
+    "ogrid",
+    "ones_like",
+    "ones",
+    "tri",
+    "tril",
+    "triu",
+    "vander",
+    "zeros_like",
+    "zeros",
+)
+
+# "core.defchararray.array",
+# "core.defchararray.asarray",
+# "core.records.array",
+# "core.records.fromarrays",
+# "core.records.fromfile",
+# "core.records.fromrecords",
+# "core.records.fromstring",
```

Comment:
Are these critical to include? I can make them appear in their own new sub-sections, but it will require slightly generalizing/complicating the code for loading objects.

Reply:
they aren't in the actual latest output table you showed me, so I removed